scripts/system/more/css/styles.css

```
@@ -8,49 +8,71 @@
     See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 */
 
+@font-face {
+    font-family: FiraSans-SemiBold;
+    src: url(FiraSans-SemiBold.ttf);
+}
+
+@font-face {
+    font-family: FiraSans-Regular;
+    src: url(FiraSans-Regular.ttf);
+}
+
+html {
+    width: 100%;
+    height: 100%;
+}
+
 body {
-    background: #27343B;
-    font-family: 'Merriweather', sans-serif;
+    background-color: #000000;
+    font-family: FiraSans-SemiBold;
```

```suggestion
    font-family: FiraSans-SemiBold, sans-serif;
```
Just in case FiraSans-SemiBold goes missing somehow. Similarly, elsewhere.

---

Fixed.